Auth/controllers/AuthController.ts

```diff
@@ -0,0 +1,24 @@
+import { AuthService } from '../services/AuthService';
+import { Request, Response } from 'express';
+import * as jwt from 'jsonwebtoken';
+
+import { privateJwtKey } from '../../config/app';
+
+
+export default class AuthController {
+	public static async loginUser(req: Request, res: Response) {
+		const { login, password } = req.query;
+		try {
+			const addedUser = await AuthService.loginUser(login, password);
```

Comment:
This is a controller. It should be as stupid as possible. So, all you need here is to get token from the AuthService and send it to the user, or catch an error from AuthService and send 401 to our user.
In other words, all the logic should be inside a service
Example of the controller (just for reference)
![image](https://user-images.githubusercontent.com/17451948/77061342-2491fe00-69eb-11ea-94f4-5adaa2f7e873.png)